### PR TITLE
Kan 36 sportplatz controller basic create und read erstellen

### PR DESCRIPTION
**Pull request by Ilimea**
validation and tests implemented

### CODE REVIEW

File: .github/workflows/dev-dotnet.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ env:
 
   # HELM env
   HELM_DIRECTORY: helm
+  HELM_VALUES_FILE: values-stage.yaml
```
Comment (mueld) [inline .github/workflows/dev-dotnet.yml:24]: why did you change this file?

**Review by mueld — CHANGES_REQUESTED** [review]
Create a new branch and open a new pull request for this branch. In this PR there are many changes to files that you should not change at all !

File: .github/workflows/dev-dotnet.yml (hunk 1 above)
Comment (Ilimea) [reply .github/workflows/dev-dotnet.yml:24]: I did not change this file manually.
I do not even know what that file is.

To me it looks like those are the commits from others whoch came in, when i pulled development branch.
